lib/datadog/appsec/metrics/collector.rb

```diff
@@ -5,14 +5,28 @@ module AppSec
     module Metrics
       # A class responsible for collecting WAF and RASP call metrics.
       class Collector
-        Store = Struct.new(:evals, :matches, :errors, :timeouts, :duration_ns, :duration_ext_ns, keyword_init: true)
+        Store = Struct.new(
+          :evals,
+          :matches,
+          :errors,
+          :timeouts,
+          :duration_ns,
+          :duration_ext_ns,
+          :input_truncated_count,
```

Comment:
WDYT of that shortcut `inputs_truncated`? Just a bit shorter

Reply:
[a744335](https://github.com/DataDog/dd-trace-rb/pull/4860/commits/a7443356ef179505e90f8e59703442b2d06cf1bc)